github/timestamp_test.go

```diff
@@ -92,7 +92,7 @@ func TestTimstamp_MarshalReflexivity(t *testing.T) {
 			t.Errorf("%s: Marshal err=%v", tc.desc, err)
 		}
 		var got Timestamp
-		err = json.Unmarshal(data, &got)
+		json.Unmarshal(data, &got)
```

Comment:
rather than removing the unused vars, how bout we actually use them by calling t.Errorf?